### ISSUE TITLE
Not working in blender 3.3?

### ISSUE THREAD

**Issue by jamesfc** [romanvolodin/dynamic_parent#8]
I tried to install it, receive no error but it doesn't show up to activate in preferences and doesn't show up in the N panel.  Assume it isn't working in 3.3?

**Comment by romanvolodin** [comment]
It's working in 3.3, I've just tried. No idea what went wrong in your case.

![Screenshot from 2022-12-02 18-02-37](https://user-images.githubusercontent.com/9511905/205322800-f2602a14-94bb-4f37-bc04-18908dd65255.png)

**Comment by jamesfc** [comment]
Thanks for looking into this - no clue here either. I tried on Blender
3.0 and it appears right away in Preferences, but on my version of
3.3, I don't receive an error and Blender log shows that it loaded and
it is in the plug-in folder, but doesn't appear in the preference
window to activate and doesn't show in the N menu. I'll continue to
play with it and see if I can figure out why.

Jim Caldwell

	-----------------------------------------From: "Roman Volodin" 
To: "romanvolodin/dynamic_parent"
Cc: "jamesfc", "Author"
Sent: Friday December 2 2022 10:06:35AM
Subject: Re: [romanvolodin/dynamic_parent] Not working in blender 3.3?
(Issue #8)

	It's working in 3.3, I've just tried. No idea what went wrong in your
case. 

	  [1] 

	—
 Reply to this email directly,  view it on GitHub [2], or  unsubscribe
[3].
 You are receiving this because you authored the thread.Message ID:  

Links:
------
[1]
https://user-images.githubusercontent.com/9511905/205322800-f2602a14-94bb-4f37-bc04-18908dd65255.png
[2]
https://github.com/romanvolodin/dynamic_parent/issues/8#issuecomment-1335384519
[3]

**Comment by jamesfc** [comment]
I discovered the problem - in 3.3 the .py file was installed in a
folder (users/app data/blender Foundation... etc.) but in 3.0 it was
installed in the plugin root directory. So in 3.3 I moved it to the
plugin root directory and it is now working!

Jim Caldwell

	-----------------------------------------From: "Roman Volodin" 
To: "romanvolodin/dynamic_parent"
Cc: "jamesfc", "Author"
Sent: Friday December 2 2022 10:06:35AM
Subject: Re: [romanvolodin/dynamic_parent] Not working in blender 3.3?
(Issue #8)

	It's working in 3.3, I've just tried. No idea what went wrong in your
case. 

	  [1] 

	—
 Reply to this email directly,  view it on GitHub [2], or  unsubscribe
[3].
 You are receiving this because you authored the thread.Message ID:  

Links:
------
[1]
https://user-images.githubusercontent.com/9511905/205322800-f2602a14-94bb-4f37-bc04-18908dd65255.png
[2]
https://github.com/romanvolodin/dynamic_parent/issues/8#issuecomment-1335384519
[3]
